test/integration/scheduler_perf/README.md

```diff
@@ -110,6 +110,12 @@ make test-integration WHAT=./test/integration/scheduler_perf KUBE_TEST_ARGS=-use
 ```
 
 Integration testing uses the same `config/performance-config.yaml` as
```

Comment:
Can you expand which labels run in the benchmark ci job?

Or maybe add a list of jobs and the label filters they use.

Reply:
Expanded the description and added a table listing the CI Jobs and their labels